src/com/sun/ts/tests/jstl/compat/onedotzero/jstl_1_0_compat_web.xml

```diff
@@ -21,7 +18,10 @@
 
 -->
 
-<web-app>
+<web-app version="5.0"
+         xmlns="https://jakarta.ee/xml/ns/jakartaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd">
```

Comment:
I'll change from `https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd` to `https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd`